### PR TITLE
Blacklist in webmail

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-7bddec051c/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-13f5e9dce3/how-to/). :bow:

File: content/rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+3. Click **Spam Settings** on the left side of the pop-up box.
+
+4. Select the third tab, which is labeled **Blacklist**.
+
+    <img src="{% asset_path rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail/spam_settings.png %}"/>
+
+5. Click **Add** in the box titled **Blacklisted Domains & Email Addresses**.
+
+6. When you are prompted with a box that is titled **Add Blacklisted Domain or Email Address**, enter the domain or email address that you wish to blacklist in the field under **Enter a domain or email address**, and then click **Add**.
```
Comment (DonNateR) [inline content/rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: I would remove... "with a box that is...". Maybe switch to, "When prompted to **Add Blacklisted Domain or Email Address**..."

File: content/rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+If you need to blacklist an IP address, follow steps 1-4 above, and continue here.
+
+1. Click **Add** in the box that is titled **Blacklisted IP Addresses**.
+
+2. When you are prompted with a box that is titled **Add Blacklisted IP Address**, enter the IP address you wish to blacklist in the field under **Enter an IP address(or range)**, and then click **Add**.
+
+    <img src="{% asset_path rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail/add_ip.png %}"/>
+
+3. Confirm that the address is now listed in the box under **Blacklisted Domains & Email Addresses**, and finally click **Save**.
```
Comment (DonNateR) [inline content/rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: Remove... "Finally".

File: content/rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    <img src="{% asset_path rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail/add_blacklist.png %}"/>
+
+7. Confirm that the address is now listed in the box under **Blacklisted Domains & Email Addresses**, and finally click **Save**.
+
+### Add a Blacklisted IP address
+
+If you need to blacklist an IP address, follow steps 1-4 above, and continue here.
+
+1. Click **Add** in the box that is titled **Blacklisted IP Addresses**.
```
Comment (DonNateR) [inline content/rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: "In the Blacklisted IP Addresses box, click **Add*."

File: content/rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+2. When you are prompted with a box that is titled **Add Blacklisted IP Address**, enter the IP address you wish to blacklist in the field under **Enter an IP address(or range)**, and then click **Add**.
```
Comment (DonNateR) [inline content/rackspace-email/blacklist-addresses-domains-and-ips-in-rackspace-email-webmail.md]: Same note as line 47.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a719b85c53/how-to/). :bow:

**Comment by catlook** [comment]
@stephamon please edit.

**Comment by stephamon** [comment]
Hey @catlook here are my suggestions. Let me know if you have any questions!

[Blacklist.addresses.domains.and.IP.addresses.in.Rackspace.Email_SFedit.docx](https://github.com/rackerlabs/rackspace-how-to/files/1468019/Blacklist.addresses.domains.and.IP.addresses.in.Rackspace.Email_SFedit.docx)
[Blacklist.addresses.domains.and.IP.addresses.in.Rackspace.Email_SFedit.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1468020/Blacklist.addresses.domains.and.IP.addresses.in.Rackspace.Email_SFedit.pdf)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d6c143fca3/how-to/)

**Comment by stephamon** [comment]
👍

**Pull request merged**